src/components/ChatRoom.tsx

```diff
@@ -126,39 +153,38 @@ function Editor({ roomId, message, onClose, ...rest }: {
   };
 
   return <>
-    <Textarea value={markdown} onChange={e => setMarkdown(e.target.value)}
+    <Textarea ref={textareaRef} value={markdown} 
+      onChange={e => setMarkdown(e.target.value)}
       autoFocus background="white" height={200} {...rest}
     />
 
-    <HStack width="100%" spacing={componentSpacing}>
-      <Button onClick={save} isLoading={saving} isDisabled={!markdown}
-        variant="brand" leftIcon={<Icon as={MdSend} />}
-      >
-        确认
-      </Button>
-      <Button onClick={() => onClose()} variant="ghost" color="grey">取消</Button>
+    <Flex direction={direction} width="100%" gap={mobileGap}>
```

Comment:
What error would you see if you use `<Flex direction={{ base: "column", [sidebarBreakpoint]: "row" }} ...>` here?

Reply:
<img width="926" alt="Screenshot 2024-09-08 at 17 12 54" src="https://github.com/user-attachments/assets/2cf5c945-793a-41ac-a794-ab8bcf78f18e">
<img width="926" alt="Screenshot 2024-09-08 at 17 13 02" src="https://github.com/user-attachments/assets/996a2f46-54eb-45bd-b007-bb897ed8e085">